generated/.tailcallrc.graphql

```diff
@@ -368,6 +368,11 @@ directive @upstream(
   """
   httpCache: Int
```

Comment:
suggested new struct might break this part 
as current definiton `@upstream(httpCache: 100)`